zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -62,15 +62,17 @@
         <li ng-if="ticket">
         <!--TODO(bzz): move to Typeahead https://angular-ui.github.io/bootstrap  -->
           <form role="search"
-            style="width: 300px; display: inline-block; margin: 0 10px"
+            style="display: inline-block; margin: 0 10px"
+             class="navbar-form"
              ng-submit="search(searchTerm)">
             <div class="input-group">
               <input
                 type="text"
                 ng-model="searchTerm"
                 ng-disabled="!navbar.connected"
                 class="form-control"
-                placeholder="Search in your notebooks"
+                tooltip-placement="bottom"
+                tooltip="Search for your Notebooks"
```

Comment:
Is there any special reason for this change? I personally prefer to have placeholder than tooltip to keep the consistency with filter input box